### ISSUE TITLE
Modal doesn't have an accessibility label

### ISSUE THREAD

**Issue by JLou** [AxaFrance/design-system#908]
This is an example of the generated HTML code by the Modal component :

```html
        <dialog
          aria-describedby="Suppression d'un bénéficiaire"
          class="af-modal"
          id="deleteBeneficiaireEnCoursDajoutModal"
          open=""
        >
          <!-- content -->
        </dialog>
```
The `aria-describedby`'s value is the `title` prop, but it should either reference the id of another element, or be `aria-label`.

We can see that no accessibility text is associated with that modal.
![Image](https://github.com/user-attachments/assets/661f43f2-f849-4e22-8313-9bb4ebbbdf63)

https://testing-playground.com/gist/56e3c8110a9e13422e2aff577def33f9/c5d9c98ffe6b119cb3600b754d30098a47dff64b

**Comment by damanV5** [comment]
@JLou  I have raised a pull request for this ^.
